packages/scaffold-config/src/component-index-template.ts

```diff
@@ -0,0 +1,28 @@
+import dedent from 'dedent'
+
+const componentIndexHtmlGenerator = (headModifier: string = '', bodyModifier: string = '') => {
```

Comment:
Let's remove the bodyModifier, no point in having it if we never use it.

Reply:
Done. Super easy to add  back later if need be.